### DISCUSSION TITLE
Is there anyway to hard core country code for phone number in pre chat form fields ?

### DISCUSSION THREAD

**Discussion by L1so** [chatwoot/chatwoot#8729] [Self hosted]
Hello, is there anyway to automatically select or hard code country code in phone number fields ? I want to automatically select Indonesia as country code without having to pick it manually. 

![Screenshot_2024-01-18_10-26-35](https://github.com/chatwoot/chatwoot/assets/44594524/d0127cad-6fcd-4325-b430-68af6f9c9496)

**Comment by malkafly** [comment]
Hey @L1so did you manage to do this?
